examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/transformers/weight_only/text-generation/README.md

```diff
@@ -111,7 +113,7 @@ Intel-extension-for-pytorch dependencies are in oneapi package, before install i
 ### Create Environment​
 Pytorch and Intel-extension-for-pytorch version for intel GPU > 2.1 are required, python version requests equal or higher than 3.9 due to [text evaluation library](https://github.com/EleutherAI/lm-evaluation-harness/tree/master) limitation, the dependent packages are listed in requirements_GPU.txt, we recommend create environment as the following steps. For Intel-exension-for-pytorch, we should install from source code now, and Intel-extension-for-pytorch will add weight-only quantization in the next version.
 
->**Note**: please install transformers==4.40.2.
+>**Note**: please install transformers==4.38.1.
```

Comment:
is this transformers version higher enough to support new models like Llama3.1-8B?  I remember it has requirements on transformers version

Reply:
We currently support Llama 3-8B, not Llama 3.1-8B.